recipes/r-catools/meta.yaml

```diff
@@ -21,7 +21,7 @@ source:
 build:
   # If this is a new build for the same version, increment the build
   # number. If you do not include this key, it defaults to 0.
-  # number: 1
+  number: 1
```

Comment:
Yes, I did that because the current build produces
"PaddingError: Placeholder of length '80' too s…"
Let me know if this is not the correct way to solve this issue as it is the first time I try solve it without updating the package version at the same time.